dev/cnf/oss_dependencies.maven

```diff
@@ -450,3 +450,13 @@ com.cloudant:cloudant-client:2.8.0
 com.cloudant:cloudant-http:2.16.0
 com.cloudant:cloudant-http:2.8.0
 wsdl4j:wsdl4j:1.6.2
+
+com.googlecode.json-simple:json-simple:1.1.1
+commons-fileupload:commons-fileupload:1.3.3
+net.sf.ehcache:ehcache-core:2.5.2
+org.apache.felix:org.osgi.service.obr:1.0.2
+org.apache.santuario:xmlsec:1.5.2
```

Comment:
This means you're pulling santuario from Maven Central? The version of Santuario that was in the com.ibm.ws.org.apache.santuario.xmlsec\lib had been manually updated, by me, in order to get WS-Security to work in Java 11. Of course, that file is long gone in favor of pulling it down from Artifactory, but I'm now I'm worried the dependencies have changed and the version of XML Security that I had updated has gone missing.

Reply:
For each of these jars, I need to do a comparison between the copy we download from Artifactory and the copy available in Maven Central. If they differ like you said santuario does, I'll upload them to DHE and Artifactory with an `-ibm` version suffix to differentiate them.

Reply:
That sounds good, if they are the same and your PB isn't having any issues on Java 11 then maybe my fix wasn't necessary. It would be great to not have a patched jar, but if they DO match and your PBs seem ok, can you run all the jaxws and wssecurity fat buckets in full mode on java 11 just to be safe? I can give you a complete list buckets to run if you need it.